bfd9000_web/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Stage 1: Builder
+FROM python:3.13-slim as builder
+
+WORKDIR /app
+
+# Install system dependencies required for building python packages
+# gcc is often needed for compiling C extensions
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    gcc \
+    python3-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install dependencies
+# We copy requirements first to leverage Docker cache
+COPY requirements.txt .
+RUN pip wheel --no-cache-dir --no-deps --wheel-dir /app/wheels -r requirements.txt
+
+# Stage 2: Final
+FROM python:3.13-slim
+
+WORKDIR /app
+
+# Install runtime dependencies
+# (Add libpq-dev here if/when we switch to Postgres)
+# libmagic1 is required for python-magic
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libmagic1 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy wheels from builder
+COPY --from=builder /app/wheels /wheels
+COPY --from=builder /app/requirements.txt .
+
+# Install packages
+RUN pip install --no-cache /wheels/*
+
+# Copy project code
+COPY . .
```

Comment:
[nitpick] can you double check that you are not putting in the docker image anything that is not needed for the docker to run? Are you sure we don´t have anything in there? If we don't, its fine.